plugins/collections/get.js

```diff
@@ -31,7 +31,7 @@ module.exports = () => ({
 
                     return Promise.all(collectionPipelines)
                         .then((pipelines) => {
-                            const result = collection.toJson();
+                            const result = Object.assign({}, collection.toJson());
```

Comment:
So I found out that I was having a bug in my tests because if I don't clone the `collection` data, then the `collection` model object gets mutated in memory. Cloning it ensures that there are no side effects from this function.